python/lsst/ctrl/mpexec/cli/utils.py

```diff
@@ -137,3 +138,20 @@ def makePipelineActions(args,
         elif args[i] in instrumentFlags:
             pipelineActions.append(_ACTION_ADD_INSTRUMENT(args[i+1]))
     return pipelineActions
+
+
+class PipetaskCommand(MWCommand):
```

Comment:
Given that the only difference between PipetaskCommand and ButlerCommand is the one string, could we set up ButlerCommand with an epilog attribute that says "See butler --help..." and then for PipetaskCommand it's a subclass of ButlerCommand that consists of a single override of the {{epilog}} value? That will stop us duplicating all the code here.

Reply:
I was worried about getting too cute trying to make them the same, but your suggestion seems like a good compromise, will do.